src/System.Data.Odbc/tests/Configurations.props

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <BuildConfigurations>
```

Comment:
Everything was failing when I added them. Now only `Vertical netfx` fails.

I've been looking and it seems the problem might be I'm using `<IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net463'">true</IsPartialFacadeAssembly>` instead of `<IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>`.

Is that the correct approach?